Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -112,6 +112,12 @@ def bump
     end
   end
 
+  sig { params(formula_or_cask: T.any(Formula, Cask::Cask), args: CLI::Args).returns(T::Boolean) }
+  def skip_repology?(formula_or_cask, args:)
+    (ENV["CI"].present? && args.open_pr? && formula_or_cask.livecheckable?) ||
```

Comment:
Do we really call `brew bump` on CI to do anything other than open PRs?

Reply:
We don't. [Other people however...](https://github.com/silver886/homebrew-tap/blob/1ec814d5af3b7c4c52cb609752e6c137629fb4cd/.github/workflows/continuous-update.yaml#L38-L40)